sdks/python/setup.py

```diff
@@ -504,6 +501,19 @@ def get_portability_package_data():
               # https://github.com/apache/beam/issues/31285
               # 'xgboost<2.0',  # https://github.com/apache/beam/issues/31252
           ],
+          'p312_ml_test': [
```

Comment:
@damccorm we should build more granular test package dependencies.

Reply:
I agree